pygmt/tests/test_solar.py

```diff
@@ -0,0 +1,157 @@
+"""
```

Comment:
@willschlitzer Nice work.

See what you wrote in CONTRIBUTING.md,

> When writing tests, don't test everything that the GMT function already tests, such as the every unique combination arguments. An exception to this would be the most popular modules, such as plot and basemap. The highest priority for tests should be the Python-specific code, such as numpy, pandas, and xarray objects and the virtualfile mechanism.

As the `solar` wrapper only has a few lines, I don't think it's necessary to have so many tests. Most of the tests should work as expected as long as the PyGMT alias mechanism works. So we don't need to have these tests, which definitely will make the testing slow.

I think we can only keep the first four tests for the 4 different day-night terminators. We may even merge these 4 tests into a single test.

Reply:
Updated the file, but moved the all of the aliases to the day-night terminator test.